### ISSUE TITLE
Thanking

### ISSUE THREAD

**Issue by ahmednasserpro** [thegenuinegourav/Android-Email-App-using-Javamail-Api#3]
i want to thank you very very much i do a lot of searching for lot days and your project the only that work successfully with me thank you very very much.
my question is that secure?

**Comment by thegenuinegourav** [comment]
Absolutely, It is secure. To make it more protective, just use getters & setters for email and password.
